docs/community/documentation-contribution-guide.rst

```diff
@@ -0,0 +1,339 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2017-2018 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+.. _documentation-contribution-guide:
+
+========================================
+Invenio Documentation Contribution Guide
```

Comment:
What do you think about `Documentation style guide` as title?

See how current title fits with the rest.
![image](https://user-images.githubusercontent.com/9035606/62197542-508b4100-b380-11e9-9b3b-a67fedb38402.png)